### PR TITLE
Add local Edinburgh and Venues info

### PR DESCRIPTION
**Pull request by rebkwok**
Local city info and venue pages.  Doesn't yet include accommodation, and needs some styling improvements, but most of the content should be here now.

### CODE REVIEW

File: _pages/location.md [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+      around 20 minutes by taxi, bus & tram. You can find more information on getting into 
+      the city from the airport on the <a href="http://www.edinburghairport.com/">airport website</a>.
+    </p>
+    <h3>By Train</h3>
+    <p>
+      There are two train stations in the centre of the city, Edinburgh Waverley, and 
+      Edinburgh Haymarket. It is approximately a 10 minute walk to the Assembly Rooms from 
+      Waverley, and 20 minutes from Haymarket. If you're travelling by train it's much 
+      cheaper to book in advance. You can book tickets online on the <a href="https://
+      thetrainline.com">Trainline</a>.
```
Comment (hypha) [inline _pages/location.md:86]: The trainline link does not work in preview. I think there might have been linebreak and whitespaces mixed in before "thetrainline.com"

File: _pages/location.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+  <div class="row column">
+    <h3>By Foot</h3>
+    <p>
+      Edinburgh is a safe and walkable city, and the main conference venue, the Assembly 
+      Rooms, is located on George Street, in the very centre of the city. 
+    </p>
+    <h3>By Bus & Tram</h3>
+    <p>
+      Edinburgh has an excellent bus and tram network.  <a href="https://www.
+      lothianbuses.com/">Lothian Buses</a> are the main city buses; the easiest way to buy tickets is with
```
Comment (hypha) [inline _pages/location.md:105]: lothian bus link doesn't work in preview

File: _pages/location.md [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+  <a href="https://www.tigerlilyedinburgh.co.uk/">Tiger Lily</a> and the 
+  <a href="https://www.candybaredinburgh.co.uk/">Candy Bar</a> to name a few.  
+  </p>
+  <p>
+  Other good pubs nearby are the 
+  <a href="https://theabbotsford.com/">Abbotsford</a> for more traditional beer styles, 
+  <a href="https://www.fiercebeer.com/bars/edinburgh-bar">Fierce Bar</a> and 
+  <a href="http://thehangingbat.com/">The Hanging Bat</a> for more modern craft beer. 
+  <a href="http://www.redsquirreledinburgh.co.uk/">Red Squirrel</a> and 
+  <a href="https://www.brewdog.com/bars/uk/lothian-road">Brewdog</a> have good pub food menus. A little further away,
```
Comment (hypha) [inline _pages/location.md:294]: the brewdog link no longer works. This does:  https://www.brewdog.com/uk/brewdog-lothian-road

File: _pages/location.md [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+  <section id="getting-here">
+  <header class="subpage-header">
+    <h2>Getting here</h2>
+  </header>
+  <div class="row column">
+    <h3>By Air</h3>
+    <p>
+      Edinburgh Airport is just 6 miles from the centre, with direct flights to many 
+      destinations around the world. You can easily get to the centre of the city in 
+      around 20 minutes by taxi, bus & tram. You can find more information on getting into
```
Comment (hypha) [inline _pages/location.md:77]: Does it make sense to specifically link the airlink buses here? I think a lot of people who are getting here by air might be interested to know that the airport buses are superbly convenient and cost-effective. You've already mentioned the tram which is also a nice option ofc.

File: _venues/caves.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+nav_id: social
+layout: base
+title: Social Event
+subtitle: The Caves
+embed_url: https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2232.7627858017763!2d-3.183719884232897!3d55.970812882624216!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4887c7f7c7c68cb1%3A0xa6d6796b12662a4a!2sThe%20Biscuit%20Factory!5e0!3m2!1sen!2suk!4v1672828017197!5m2!1sen!2suk
```
Comment (hypha) [inline _venues/caves.md:6]: The embed_url shows the Biscuit Factory's link.

File: _pages/location.md [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+    <li><a href="http://www.thecityrestaurant.co.uk/">City Restaurant</a> - cheap, friendly and relaxed, and a favourite with students.</li>
+    <li><a href="https://www.thefishmarketnewhaven.co.uk/">The Fishmarket</a> - a little 
+    further away on the coast in Newhaven but well worth a visit for the freshest possible 
+    fish & chips, sourced from the excellent 
+    <a href="https://www.welchfishmongers.com">Welch Fishmongers</a> next door.</li>
+    </ul>  
+  </p>
+  <h3>Burgers</h3>
+    <p>
+    You'll find a few branches of burger chains such as <a href="https://www.google.com/maps/place/fountain+park,+Dundee+St,+Edinburgh+EH11+1AF,+UK/@55.9414843,-3.2192654,17z/data=!3m1!4b1!4m5!3m4!1s0x4887c7a6567bf729:0x58fdfbfbb24837d0!8m2!3d55.9416438!4d-3.2168058">Five Guys</a> around Edinburgh,
```
Comment (hlwill) [inline _pages/location.md:254]: This link gave me a pin in Fountain Park, so might not be what we want

**Comment by rebkwok** [comment]
@hypha @hlwill Thanks for checking all the links! They should all be fixed now, and I've added links for the airport buses/trams

**Comment by hlwill** [comment]
Awesome - it looks good to me :+1: 
I think we'll need someone with permissions to merge it

**Comment by hypha** [comment]
LGTM too! And thanks, @rebkwok for the lovely content! You've certainly convinced me to get out more!

**Pull request merged**